### ISSUE TITLE
Should be able to click on Submit button with click("Submit")

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#312]
**Expected behavior**
Should be able to click on Submit button with click("Submit")

**Actual behavior**
When an input element with type submit is used the UI is rendered with a button with text "Submit". click("Submit") says `Error: Element with text Submit not found, run `.trace` for more info.`

**Steps to replicate**
* Create a html file
```
<html>
<body>
​
<form action="/action_page.php">
  <input type="submit">
</form>
​​
</body>
</html>
```
* Open the REPL
```
> openBrowser()
 ✔ Browser opened
> goto("file:///Users/sswaroop/Desktop/newFile.html")
 ✔ Navigated to url "file:///Users/sswaroop/Desktop/newFile.html"
> click("Submit")
 ✘ Error: Element with text Submit not found, run `.trace` for more info.
```
> Here what is visible to the user is the text "Submit" however in the DOM the input is type="submit"

**Comment by NivedhaSenthil** [comment]
Fix available in fb02d59

**Comment by sguptatw** [comment]
Issue replicable with the same steps as original issue with commit - 0de7dcadd9efdeede68c4493c5a4db2a2c111088

**Comment by NivedhaSenthil** [comment]
Text made case insensitive now. Fix available in 99fca06

**Comment by sguptatw** [comment]
Verified on version `5ca341b78e2c14c2347206209d15df79955699bc`
